[2012-11-08 11:52:54] request.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /sitio/privacidad" (uncaught exception) at /www/htdocs/jlozano/cupon/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Bundle\FrameworkBundle\EventListener\SessionListener::onKernelRequest". [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelRequest". [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException". [] []
[2012-11-29 00:53:19] request.CRITICAL: Symfony\Component\Yaml\Exception\ParseException: Malformed inline YAML string { _controller: FrameworkBundle:Redirect:redirect, in "\/home\/jlozano\/Proyectos\/Cupon\/app\/config\/routing.yml" at line 31 (near "defaults: { _controller: FrameworkBundle:Redirect:redirect,") (uncaught exception) at /home/jlozano/Proyectos/Cupon/vendor/symfony/symfony/src/Symfony/Component/Yaml/Inline.php line 352 [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Bundle\FrameworkBundle\EventListener\SessionListener::onKernelRequest". [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelRequest". [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelRequest". [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\Security\Http\Firewall::onKernelRequest". [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ControllerListener::onKernelController". [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener::onKernelController". [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\TemplateListener::onKernelController". [] []
[2012-11-29 00:53:19] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException". [] []
[2012-11-29 00:53:19] request.CRITICAL: Exception thrown when handling an exception (ErrorException: Warning: file_get_contents(/www/htdocs/jlozano/cupon/vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception_full.html.twig) [<a href='function.file-get-contents'>function.file-get-contents</a>]: failed to open stream: No such file or directory in /home/jlozano/Proyectos/Cupon/vendor/twig/twig/lib/Twig/Loader/Filesystem.php line 126) [] []
